querybook/config/querybook_default_config.yaml

```diff
@@ -13,6 +13,9 @@ REDIS_URL: ~
 ELASTICSEARCH_HOST: ~
 ELASTICSEARCH_CONNECTION_TYPE: naive
 
+# --------------- Lineage ---------------
+DATA_LINEAGE_BACKEND: logic.lineage
```

Comment:
can you have the `DATA_LINEAGE_BACKEND` to be in something like lib.lineage.db? In there you can just call the logic.lineage functionality as is